.rspec

```diff
@@ -1,3 +1,2 @@
 --warnings
 --color
---require spec_helper
```

Comment:
Our convention is that `spec_helper` contains the bare minimum required to configure the spec suite and everything else assumes its presence